src/index.ts

```diff
@@ -58,13 +54,23 @@ app.command("/treeumph", async ({ command, ack, say }) => {
 
 app.message("New shoutout from", async ({ message, say }) => {
   const botMessage = message as BotMessageEvent;
-  console.log(botMessage);
-  if (botMessage.user === BOT_USER_ID) {
-    const res = await plantTree();
-    if (res) {
+  if (botMessage.user === HR_BOT_USER_ID) {
+    const count = await getEstimatedMonthlyTreeCount();
+    if (count <= MONTHLY_TREE_BUDGET) {
```

Comment:
Is the tree budget limited at the API side? If so, do we know what response we get if we try and plant a tree that would go over the budget? 

I'm wondering if it would be easier to just make the request and handle the failure, rather than having to do a convoluted search of Slack messages.

Reply:
Nah I meant to test that, think I've got a few credits left before I run out and can give it a go. I'll see how finance wanna  fund the account and maybe we can do away with all this searching if it's gonna be monthly and we expect to run out. 

I'll add a todo for this and come back to it based on what finance say.

Reply:
This might make it easier to support if they ever change their pricing too, e.g. what if it becomes $2/tree

Reply:
@velvetkevorkian Managed to do away with this counting by checking how many credits we have left in the account.